File: frontend/scripts/wipe_db.ts
import { prisma } from '../src/lib/prisma'

async function main() {
  console.log("Wiping database tables...")
  await prisma.constraint.deleteMany()
  await prisma.extraction.deleteMany()
  await prisma.aIJob.deleteMany()
  await prisma.documentChunk.deleteMany()
  await prisma.document.deleteMany()
  await prisma.project.deleteMany()
  console.log("Database wiped successfully! (Kept User and Organization)")
}

main()
  .catch(e => {
    console.error(e)
    process.exit(1)
  })
  .finally(async () => {
    await prisma.$disconnect()
  })

File: backend/wipe_db.py
import os
import psycopg
from dotenv import load_dotenv

load_dotenv()
db_url = os.getenv("DATABASE_URL").split("?")[0]

with psycopg.connect(db_url) as conn:
    with conn.cursor() as cur:
        cur.execute('TRUNCATE TABLE "Constraint", "Extraction", "AIJob", "DocumentChunk", "Document", "Project", "User", "Organization" CASCADE;')
        conn.commit()
print("Wiped ALL data successfully.")
